docs/source/automating-flow-runs.rst

```diff
@@ -0,0 +1,70 @@
+Running a flow automatically
+****************************
+
+It is frequently desirable to run a flow when new data becomes available.
+This document provides examples for how to accomplish this
+using `watchdog`_ to monitor for filesystem events
+and using the Globus Automate CLI to run already-defined flows.
+
+
+Shell scripting
+===============
+
+`watchdog`_ provides a CLI tool named ``watchmedo``.
+If you're comfortable with shell scripting
+then you can use the following command to run a script named ``runner.sh``.
+
+
+..  literalinclude:: code_snippets/watchmedo.sh
+    :caption: ``watchmedo.sh`` [:download:`download <code_snippets/watchmedo.sh>`]
```

Comment:
Is the `:download:` role coming from a sphinx extension? I haven't seen this before but it seems _really nice_.

Reply:
I _think_ it's a built-in role...but I'm not sure!

Reply:
[Looks](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html) like it is! I must have missed it when it was added... before I started using sphinx... 😅 
![image](https://user-images.githubusercontent.com/1300022/179273991-f02d7e49-5577-4047-8d51-1ef16c33a5af.png)